### PR TITLE
fix content feedback trigger

### PR DESCRIPTION
**Pull request by rudigiesler**
This won't only occur on new sessions, this might occur in exiting sessions, so we need to move it to the process_message function

### CODE REVIEW

**Review by KaitCrawford — COMMENTED** [review]
@rudigiesler are all the changes to tests necessary because "1" and "2" are in the feedback keywords? Is there any way we can avoid that?

File: yal/onboarding.py [hunk 1]
```diff
@@ -97,17 +97,15 @@ async def state_save_persona_emoji(self):
...
+        msg = self._(
+            "\n".join(
+                [
+                    "*[persona_emoji] Great! I'm just going to ask you a few "
+                    "quick questions now.*",
+                ]
+            )
         )
-        self.publish_message(msg)
+        await self.publish_message(msg)
```
Comment (rudigiesler) [inline yal/onboarding.py:108]: Unrelated, but it was showing a warning, so I fixed it

**Comment by rudigiesler** [comment]
@KaitCrawford Yes, `1`, `2`, and `yes` are fairly common responses. We could possible remove `yes`, because that isn't explicitly mentioned, but `1` and `2` are in the content of the message sent to the user, so the only way we can avoid that is to change the content.

**Comment by KaitCrawford** [comment]
> @KaitCrawford Yes, `1`, `2`, and `yes` are fairly common responses. We could possible remove `yes`, because that isn't explicitly mentioned, but `1` and `2` are in the content of the message sent to the user, so the only way we can avoid that is to change the content.

bah humbug! ☹️

**Review by KaitCrawford — APPROVED** [review]
👍

**Pull request merged**